.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 dist
+test
```

Comment:
why ignoring tests for linting?

Reply:
+1.
I'll update

Reply:
And I will update version of @typescript-eslint